project/Dependencies.scala

```diff
@@ -20,15 +21,15 @@ object Dependencies {
     "specs2-core",
     "specs2-junit",
     "specs2-mock"
-  ).map("org.specs2" %% _ % specsVersion)
+  ).map("org.specs2" %% _ % specsVersion cross CrossVersion.for3Use2_13)
 
   val slf4jtest = Seq("uk.org.lidalia" % "slf4j-test" % "1.2.0")
 
   val junitInterface = Seq("com.novocode" % "junit-interface" % "0.11")
 
   val scalaJava8Compat = Seq("org.scala-lang.modules" %% "scala-java8-compat" % "1.0.0")
 
-  val playJson = Seq("com.typesafe.play" %% "play-json" % "2.9.2")
+  val playJson = Seq("com.typesafe.play" %% "play-json" % "2.9.2" cross CrossVersion.for3Use2_13)
```

Comment:
There is play-json 2.10.0-RC5 which gets cross build for Scala 3 already: https://repo1.maven.org/maven2/com/typesafe/play/play-json_3/